frontend/src/metabase/visualizations/components/ClickActions/ClickActionControl.module.css

```diff
@@ -0,0 +1,33 @@
+button.horizontalButton {
+  display: flex;
+  flex: auto;
+  align-items: center;
+  border-radius: 8px;
+  border: none;
+  text-align: start;
+  padding: 0.5rem;
+  margin: 0 -0.5rem;
+  width: auto;
+  min-width: 148px;
+  line-height: 1rem;
+  color: var(--color-text-dark);
+
+  &:hover {
+    color: var(--color-white);
+    background-color: var(--mb-color-brand);
+
+    .nested {
+      color: var(--color-white);
+    }
+  }
+
+  .inner {
+    width: 100%;
+  }
+
+  .label {
```

Comment:
This is one of the cases I discussed in the FE guild.

I need the button content to render as flex in this one usecase, do I need to override all Buttons everywhere on the site for that?

Reply:
Is this the same as https://metaboat.slack.com/archives/C057WD5L0JG/p1715204376544389?

Reply:
It's a similar issue than in that thread, but not the same exact component.

`ClickActionControl` is using some deprecated components and we're adding new designs and features to it.

Reply:
But you're overrides styles for a button in metabase/ui, which is new

Reply:
True, I meant more that this seems like a very one of usecase for the button.

I'm happy to make it a custom button, so there are no overrides, but I feel like that's not the way to go either.

The only reason I switched to `ui/Button` was because the legacy button that was used could not be customized properly (without having a large blast radius).

But I understand the comment to be about making sure we streamline the design process into forcing a more principled approach.

I'm not sure what the next steps here should be: push back to the design team? Use a fully custom component instead of using overrides?

Reply:
Could you ask @kdoh in ui proj channel?